android/src/main/java/com/imagepicker/Utils.java

```diff
@@ -320,6 +322,24 @@ static boolean isValidRequestCode(int requestCode) {
         }
     }
 
+    public static class VideoMetadata {
+        int duration;
+        int bitrate;
+
+        public VideoMetadata(int duration, int bitrate) {
+            this.duration = duration;
+            this.bitrate = bitrate;
+        }
+
+        public int getBitrate() {
+            return bitrate;
+        }
+
+        public int getDuration() {
+            return duration;
+        }
+    }
```

Comment:
Do we think it would be worth separating this out into its own file? `/imagepicker/video-metadata.java` ?